src/pages/checkout-page/checkbox-form/checkbox-form.tsx

```diff
@@ -0,0 +1,48 @@
+import { Box, Checkbox, FormControlLabel } from "@mui/material";
+import { ChangeEvent, FC, useState } from "react";
+import styled from "styled-components";
+
+const FormCheckBox = styled(Box)`
+  padding: 15px 0;
+`;
+
+const CheckboxForm: FC = () => {
+  const [checkedOne, setCheckedOne] = useState<boolean>(false);
```

Comment:
Don't use verbs as variables names. Use nouns, here could be firstCheckbox...

Reply:
The entire checkout component needs to be reworked.